app/mailers/activity_mailer.rb

```diff
@@ -5,16 +5,20 @@
 class ActivityMailer < ApplicationMailer
   helper ApplicationFormattersHelper
 
+  layout :choose_layout
+
   # Emails a recipient, informing him of an activity.
   #
   # @param [User] recipient The recipient of the email.
   # @param [Course::Notification|UserNotification] notification The notification to be made
   #   available to the view, accessible using +@notification+.
   # @param [String] view_path The path to the view which should be rendered.
-  def email(recipient, notification, view_path)
+  # @param [String] layout_path The filename in app/views/layouts which should be rendered.
```

Comment:
Suggest to document the default behaviour if layout_path is not specified.

Reply:
Now documented on the next line of the comment.